services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ProtectedPathSpec.groovy

```diff
@@ -752,22 +757,43 @@ class ProtectedPathSpec extends BaseSpecification {
         given: "Two active not neighboring switches with two possible paths at least"
         def (Switch srcSwitch, Switch dstSwitch) = getNotNeighboringSwitchPair(2)
 
-        and: "Create a flow with protected path"
+        and: "Create a flow without protected path"
         def flow = flowHelper.randomFlow(srcSwitch, dstSwitch)
-        flow.allocateProtectedPath = true
+        flow.allocateProtectedPath = false
         flowHelper.addFlow(flow)
-        def flowPathInfo = northbound.getFlowPath(flow.id)
+        assert !northbound.getFlowPath(flow.id).protectedPath
         Wrappers.wait(WAIT_OFFSET) { assert northbound.getFlowStatus(flow.id).status == FlowState.UP }
+
+        and: "Cookies created by flow"
+        def createdCookies = northbound.getSwitchRules(srcSwitch.dpId).flowEntries.findAll {
+            !Cookie.isDefaultRule(it.cookie)
+        }*.cookie
+        assert createdCookies.size() == 2
+
+        when: "Update flow: disable protected path(allocateProtectedPath=true)"
```

Comment:
are you sure this description is correct?

Reply:
you right, I forgot to rename it.
fixed